bin/tomcat-ctl

```diff
@@ -11,187 +11,160 @@ echo_time() {
+            exit 1
+               ;;
+    esac
+
+    # Download the WAR file if it doesn't exist
+    WAR_FILE="$TOMCAT_KFS_DIR/kfs.war"
+    log "Looking for $WAR_FILE..."
+    echo_time "Looking for $WAR_FILE..."
+    if [ ! -f $WAR_FILE ]
+    then
+        MAVEN_SERVER="https://ka-tools.mosaic.arizona.edu/nexus"
+        MAVEN_ENDPOINT="$MAVEN_SERVER/service/local/artifact/maven/redirect"
+        WAR_SOURCE_URL="$MAVEN_ENDPOINT?r=$APP_REPO&g=org.kuali.kfs&a=kfs-web&v=$APP_VERSION&c=ua-ksd&p=war"
+        curl -L "$WAR_SOURCE_URL" -o $WAR_FILE
+        WAR_URL_FOR_PRINT=`curl -Is $WAR_SOURCE_URL | awk '/Location/ { print $2 }'`
+        log "WAR file downloaded: $WAR_URL_FOR_PRINT"
+        echo_time "WAR file downloaded: $WAR_URL_FOR_PRINT"
+        chown kualiadm:kuali $WAR_FILE
+        # Extract the WAR file
+        cd $TOMCAT_KFS_DIR
+        jar -xf $WAR_FILE
     fi
 
+    log "Starting Tomcat application server tomcat ..."
+    echo_time "Starting Tomcat application server tomcat ..."
+
+    # copy in tomcat configuration files
+    log "Copying tomcat configuration files."
+    echo_time "Copying tomcat configuration files."
+    cp $TOMCAT_CONFIG_DIRECTORY/server.xml $TOMCAT_BASE_DIR/conf/
+    cp $SECURITY_DIRECTORY/tomcat-users.xml $TOMCAT_BASE_DIR/conf/
+    cp $TOMCAT_CONFIG_DIRECTORY/logging.properties $TOMCAT_BASE_DIR/conf
+    cp $TOMCAT_CONFIG_DIRECTORY/log4j.properties $TOMCAT_KFS_WEBINF_DIR/classes
+
+    # copy in New Relic config file
+    log "Copying New Relic configuration file."
+    echo_time "Copying New Relic configuration file."
+    cp $SECURITY_DIRECTORY/newrelic.yml $TOMCAT_SHARE_LIB/
+
+    # copy in oracle JDBC drivers
+    log "Copying oracle JDBC drivers."
+    echo_time "Copying oracle JDBC drivers."
+    cp $TOMCAT_CONFIG_DIRECTORY/classes/* $TOMCAT_SHARE_LIB/
+
+    # Get changelogs for liquibase
+    log "Loading changelog files for liquibase."
+    echo_time "Loading changelog files for liquibase"
+
+	# create new directory to hold the files extracted from kfs-core-ua jar file
+	mkdir -p $TOMCAT_KFS_DIR/changelogs/
+
+	# cd to the new directory for changelog files
+	cd $TOMCAT_KFS_DIR/changelogs/
+
+	# extract files in kfs-core-ua* jar
+	/usr/bin/jar -xf $TOMCAT_KFS_DIR/WEB-INF/lib/kfs-core-ua*
+
+	# run liquibase here
+    log "Running liquibase update."
+    echo_time "Running liquibase update."
+	liquibase_update_kfs.sh $APP_VERSION
+
+    cp $TOMCAT_CONFIG_DIRECTORY/setenv.sh /usr/share/tomcat7/bin/
+    chown kualiadm:kuali $TOMCAT_BASE_DIR/bin/setenv.sh
+    chmod +x $TOMCAT_BASE_DIR/bin/setenv.sh
+
+    /usr/share/tomcat7/bin/catalina.sh start
+
+    # Tail the tomcat log file to keep the container alive
+    tail -f /logs/catalina.out
+
     return;
 }
 
 tomcat_stop() {
+
+    log "Stopping Tomcat application server tomcat ..."
+    echo_time "Stopping Tomcat application server tomcat ..."
+
+    /usr/share/tomcat7/bin/catalina.sh stop
+
+    sleep 3
     if check_process
```

Comment:
It looks like you still are calling on the check_process method that you removed from the top.